### ISSUE TITLE
Penpot Standalone Package viz. Appimage

### ISSUE THREAD

**Issue by zenny** [penpot/penpot#767]
Hi,

It appears to be a wonderful concept for prototyping, but very clumpsy in deploying (for the paranoids) ;-)

How can one deploy Penpot as an standalone application without using any complicated tools like Docker or requiring to login except LDAP?

It would be wonderful if penpot is penpot into formats like an appimage, flatpak or a package (deb, rpm, apk among others) for local deployment!

Cheers, and stay safe.

**Comment by hirunatan** [comment]
Yes, I agree with you in that it's important to have many different ways of installation.

It's in our roadmap to have this as soon as possible, although perhaps we'll wait a bit more, because in this Alpha stage we are still making big changes on the deployment setup, and it's complicated to maintain if we have many packaging and distribution formats.

At some point we'll write an Ansible role with the detailed steps to build a Penpot instance. This would serve as a basis for building linux distribution packages, or packaged apps for AppImage, Yunohost, Sandstorm or Cloudron, for example.

**Comment by zenny** [comment]
@hirunatan Thanks for the update and the hard work you are onto. I came to learn about you from the presentation at FOSDEM2021 and overwhelmed with the approach better than other prototyping tools like pencil.

However, it would be nice if there are some hints on deploying Penpot without docker, say in *BSD. 

Look forward to. (BTW, this issue deserves to be open to get more inputs from the community rather than an abrupt closure, imho)

Cheers.

**Comment by hirunatan** [comment]
Well, I can leave it open if you want. The closing is not definitive, it's just delayed. But it's in the roadmap, so let's keep it open here too.

**Comment by SpeeQz1** [comment]
Is there any update on the implementation of a standalone package in the form of an AppImage?
